desktop/core/src/desktop/lib/fsmanager.py

```diff
@@ -117,8 +117,13 @@ def _get_client_cached(fs, name, user):
   global CLIENT_CACHE
   if CLIENT_CACHE is None:
     CLIENT_CACHE = {}
-  # We don't want to cache by username when IDBroker not enabled
-  cache_key = _get_cache_key(fs, name, user) if conf_idbroker.is_idbroker_enabled(fs) else _get_cache_key(fs, name)
+
+  # We don't want to cache by username when IDBroker or RAZ is not enabled
```

Comment:
Comment reads a bit odd. Cleaner would be "Only cache by username when IDBroker or Raz is enabled". But... It doesn't actually add any value so might as well remove it.

Reply:
Changing it